common/reviews/api/heft-config-file.api.md

```diff
@@ -33,6 +35,17 @@ export interface IJsonPathsMetadata {
     [jsonPath: string]: IJsonPathMetadata;
 }
 
+// @beta (undocumented)
+export interface ILoadConfigurationFileOptions {
+    ignoreIfNotExist?: boolean;
+}
+
+// @beta (undocumented)
+export interface ILoadConfigurationFileOptionsIgnoreNotExist extends ILoadConfigurationFileOptions {
```

Comment:
Seems like an odd interface to export for such little utility

Reply:
Yeah I'm going to clean this up a bit.